linux/network_device_ops/prepare_network_device_test.yml

```diff
@@ -35,17 +48,25 @@
         package_list: ["rdma-core", "librdmacm-utils"]
         package_state: "present"
       when: guest_os_family in ["Suse", "RedHat"]
-  when: adapter_type == "pvrdma"
 
-- name: "Get existing network interface before hotadd testing"
-  include_tasks: ../utils/get_network_adapters_status.yml
-
-- name: "Set fact of network adapter info before hotadd"
-  ansible.builtin.set_fact:
-    network_adapters_before_hotadd: "{{ guest_network_adapters }}"
+    - name: "Get network config file for '{{ eth0_name }}'"
+      include_tasks: ../utils/get_network_config_file.yml
+      vars:
+        network_adapter_name: "{{ eth0_name }}"
 
-# Get eth0 device name in guest. Sometimes it is not 'eth0',
-# e.g., 'ens192', 'ens33', etc.
-- name: "Set fact of the first network adapter interface name in guest OS"
-  ansible.builtin.set_fact:
-    eth0_name: "{{ network_adapters_before_hotadd[0] }}"
+    - name: "Remove MAC address or UUID from network config file"
+      block:
+        - name: "Get network config file state"
+          include_tasks: ../utils/get_file_stat_info.yml
+          vars:
+            guest_file_path: "{{ network_config_path }}"
+    
+        - name: "Remove MAC address or UUID from network config file for '{{ eth0_name }}'"
```

Comment:
This yml 'prepare_network_device_test.yml" is executed at the very beginning of the case. Will it impact the IP address when remove the MAC/UUID from the network config file? Or will the guest OS update the MAC/UUID to the file again automatically?

Reply:
No, it won't.